pkg/telemetry/prometheus/packets.go

```diff
@@ -155,6 +163,12 @@ func initPacketStats(nodeID string, nodeType livekit.NodeType) {
 		Name:        "jitter",
 		ConstLabels: prometheus.Labels{"node_id": nodeID, "node_type": nodeType.String()},
 	})
+	promAsnBytes = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace:   livekitNamespace,
+		Subsystem:   "packet",
+		Name:        "asnBytes",
```

Comment:
this naming is not according to Prometheus conventions.

Reply:
Fixed.